rasa/core/domain.py

```diff
@@ -127,7 +128,7 @@ def warn_missing_templates(
                 message += (
                     "\nUtterance '{}' is listed as an "
                     "action in the domain file, but there is "
-                    "no matching utterance template.   Please "
+                    "no matching utterance template. Please "
                     "check your domain."
```

Comment:
This warning also shows up twice with `rasa train`, I think because of the CLI question i have above

Reply:
yep, should be related. will test it in a bit.